fe/fe-core/src/main/java/com/starrocks/statistic/StatisticAutoCollector.java

```diff
@@ -69,94 +55,139 @@ protected void runAfterCatalogReady() {
 
         initDefaultJob();
 
-        List<TableCollectJob> allJobs = generateAllJobs();
+        updateHealthy();
 
-        collectStatistics(allJobs);
+        List<BaseCollectJob> allJobs;
+        if (Config.enable_collect_full_statistics) {
+            allJobs = generateFullAnalyzeJobs();
+        } else {
+            allJobs = generateAllJobs();
+        }
+
+        for (BaseCollectJob tcj : allJobs) {
+            statisticExecutor.collectStatistics(tcj);
+        }
 
         expireStatistic();
     }
 
     private void initDefaultJob() {
-        // Add a default sample job if was't been collect
+        // Add a default sample job if wasn't collect
         List<AnalyzeJob> allAnalyzeJobs = GlobalStateMgr.getCurrentAnalyzeMgr().getAllAnalyzeJobList();
         if (allAnalyzeJobs.stream().anyMatch(j -> j.getScheduleType() == ScheduleType.SCHEDULE)) {
             return;
         }
 
         AnalyzeJob analyzeJob = new AnalyzeJob(AnalyzeJob.DEFAULT_ALL_ID, AnalyzeJob.DEFAULT_ALL_ID, Collections.emptyList(),
                 AnalyzeType.SAMPLE, ScheduleType.SCHEDULE, Maps.newHashMap(), ScheduleStatus.PENDING, LocalDateTime.MIN);
-
         GlobalStateMgr.getCurrentAnalyzeMgr().addAnalyzeJob(analyzeJob);
     }
 
-    private void collectStatistics(List<TableCollectJob> allJobs) {
-        // AnalyzeJob-List<TableCollectJob> for update AnalyzeJob status
-        Map<AnalyzeJob, List<TableCollectJob>> analyzeJobMap = Maps.newHashMap();
+    private void updateHealthy() {
```

Comment:
The Healthy word seem strange in Statistic Collected context.

Reply:
change the word to "statisticsHealthy"?